flexmeasures/data/services/forecasting.py

```diff
@@ -117,7 +117,7 @@ def create_forecasting_jobs(
     return jobs
 
 
-def make_forecasts(
+def make_rolling_viewpoint_forecasts(
```

Comment:
Please let the docstring explain shortly what it means to roll the viewpoint.

To quote timetomodel: "The time window of the specs (training + test data) is moved forwards also step by step." That seems helpful.

I'm not sure if that is also all that would be needed to set this apart from a fixed-viewpoint forecast, though.